docs/how-to/apphook_configurations.rst

```diff
@@ -0,0 +1,110 @@
+.. _multiple_news_sections:
+
+#############################
+Multiple news/weblog sections
+#############################
+
+Your project can host multiple independent news/weblog sections, each with their own items. For
+example, a company website might have a news section for publishing press releases, and a weblog for
+publishing more informal articles.
+
+To do this, you need to create a django CMS page for each news/weblog section, and add an apphook
+for Aldryn News and Blog to each of them. You will also need to create a separate apphook
+configuration for each of them - apphook configurations cannot be shared between apphook instances.
+
+.. note:
+
+    Creating a new News & Blog section on your site implies setting up a new apphook instance.
+    The apphook instance however doesn't actually do anything until a page has been set up with it.
+
+
+**********************************
+Creating a new News & Blog section
+**********************************
+
+The quickest way to do this is:
+
+* Create the new page.
+* In its *Advanced settings*, choose *Newsblog* in the *Application* field. A new field,
+  *Application configurations*, will appear immediately below it.
+* Add a new application configuration, by selecting the **+** icon.
+
+
+Fields
+======
+
+*Instance namespace* - a unique name for this configuration. Note that this cannot be subsequently changed.
```

Comment:
Perhaps a note that this should be slug-like in that it shouldn't contain spaces, but can use underscores or dashes to separate words.